nubilanding/src/components/beneficios/beneficios.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import styles from './styles.module.scss';
+ 
+function  Beneficios() {
+    return(
+            <div className= {styles.contenedorGeneral}>
```

Comment:
Cuidado que a partir de está línea se está identando con 4 espacios de más
```suggestion
        <div className= {styles.contenedorGeneral}>
```